Dockerfile

```diff
@@ -14,9 +14,8 @@ WORKDIR /tmp/build
 RUN \
     install_packages \
             build-essential \
-            libdbus-glib-1-dev
-
-RUN pip3 install --no-cache-dir --target="$PYTHON_DEPENDENCIES_DIR" .
```

Comment:
actually this might make it worse. Every RUN results in a docker fs layer. apt install isn't likely to change very often while pip3 install might change more often. It is a good practice to club only related ones together.

Reply:
Well, Docker Linting was whining about it constantly. So I've gathered them. Now looks peaceful :smile: